src/model/view/index.js

```diff
@@ -1,55 +1,105 @@
 import { ValuesFridge, ValuesReport } from 'model/data/fridge/yup.mjs';
 
-const fridgeCache = {};
+// in memory cache
+const cacheViewFridgeList = [];
+const viewFridgeFor = {};
 
 export async function getFridgeList() {
-  let fridgeList = Object.values(fridgeCache);
-  if (fridgeList.length == 0) {
+  if (cacheViewFridgeList.length === 0) {
     await fetchAllData();
-    fridgeList = Object.values(fridgeCache);
   }
-  return fridgeList;
+  return cacheViewFridgeList;
 }
 
-export async function getGhostFridgeList() {
-  return fetch('data/ghostFridges.json').then((response) => response.json());
+const sortByNameAsc = (a, b) => {
+  const nameA = a.name;
+  const nameB = b.name;
+  if (nameA < nameB) {
+    return -1;
+  }
+  if (nameA > nameB) {
+    return 1;
+  }
+  return 0;
+};
+
+const castOptions = Object.freeze({ stripUnknown: true }); // yup configuration
+
+function viewFridgeFromLocal(apiFridge) {
+  const viewFridge = ValuesFridge.cast(apiFridge, castOptions);
+  viewFridge['report'] = null;
+  return viewFridge;
+}
+
+function viewFridgeFromRemote(apiFridge) {
+  const report = apiFridge.latestFridgeReport ?? null;
+  const viewFridge = ValuesFridge.cast(apiFridge, castOptions);
+  viewFridge['report'] = ValuesReport.cast(report, castOptions);
+  return viewFridge;
 }
 
+function loadIntoCache({ fridges }, fnConverter) {
```

Comment:
I remember we had an issue with the cache during launch, excited to test this out

Reply:
The complete API request code has not been integrated yet. I'll do it gradually as we clean up the codebase. The following files have hotfix code that is not part of this PR:

- M src/components/atoms/FeedbackCard/FeedbackCard.jsx
- M src/components/atoms/PageFooter/PageFooter.jsx
- M src/components/molecules/AppBar/AppBar.jsx
- M src/components/molecules/FridgeInformation/FridgeInformation.jsx
- M src/components/molecules/index.js
- A src/components/organisms/DialogFridgeReport/PanelConfirm.jsx
- M src/components/organisms/browse/List.jsx
- M src/components/organisms/browse/components/MapMarkerList.jsx
- M src/components/organisms/browse/model/markersFrom.js
- A src/components/organisms/dialog/CreateFridge.jsx
- M src/components/organisms/dialog/components/PanelFridge.jsx
- M src/components/organisms/dialog/components/PanelMaintainer.jsx
- M src/components/organisms/dialog/components/PanelReport.jsx
- A src/components/organisms/dialog/components/prop-types.js
- D src/components/organisms/index.js
- M src/pages/fridge/[id].jsx
- R src/pages/user/fridge/add.jsx	src/pages/user/add-fridge.jsx
- D src/pages/user/fridge/report.jsx
- A src/pages/user/fridge/report/[fridgeId].jsx